app/assets/javascripts/discourse/app/components/category-logo.js

```diff
@@ -0,0 +1,4 @@
+import Component from "@ember/component";
+
+// TODO: Remove in December 2021
```

Comment:
Can you try a template only Glimmer component here?

Basically it would be to replace the contents of this file with: 

```
import templateOnly from "@ember/component/template-only";
export default templateOnly();
```

Glimmer is the new way Ember components are done.